src/Lab/Triangle/Triangle.csproj

```
@@ -9,10 +9,10 @@
     </PropertyGroup>
 
     <ItemGroup>
-      <ProjectReference Include="..\..\Input\Silk.NET.Input.Common\Silk.NET.Input.Common.csproj" />
-      <ProjectReference Include="..\..\Input\Silk.NET.Input\Silk.NET.Input.csproj" />
+      <!--<ProjectReference Include="..\..\Input\Silk.NET.Input.Common\Silk.NET.Input.Common.csproj" />
+      <ProjectReference Include="..\..\Input\Silk.NET.Input\Silk.NET.Input.csproj" />-->
```

?

---

Was for testing purposes. Will fix.
```suggestion
      <ProjectReference Include="..\..\Input\Silk.NET.Input\Silk.NET.Input.csproj" />
```